lib/msf/java/rmi/client.rb

```diff
@@ -93,14 +101,16 @@ def recv_protocol_ack(opts = {})
         #
         # @param opts [Hash]
         # @option opts [Rex::Socket::Tcp] :sock
-        # @return [Rex::Java::Serialization::Stream]
+        # @return [Rex::Proto::Rmi::Model::ReturnValue] if success
+        # @return [NilClass] otherwise
         # @see Rex::Proto::Rmi::Model::ReturnData.decode
         def recv_return(opts = {})
           nsock = opts[:sock] || sock
           data = safe_get_once(nsock)
+
           begin
             return_data = Rex::Proto::Rmi::Model::ReturnData.decode(StringIO.new(data))
-          rescue ::RuntimeError
+          rescue ::RuntimeError => e
```

Comment:
No need to store it to a variable if you are not going to do anything with it anyway.  Perhaps a better question, though, is why is this `RunTimeError`? It seems like the `ReturnData.encode` above should raise a class-specific exception instead.